lib/findbugs_translate_checkstyle_format/translate.rb

```diff
@@ -22,7 +22,13 @@ def trans(xml)
 
       bug_instances = [bug_instances] if bug_instances.is_a?(Hash)
       bug_instances.each do |bug_instance|
-        source_lines = bug_instance['SourceLine']
+        source_lines = if bug_instance.has_key?('SourceLine')
+          bug_instance['SourceLine']
```

Comment:
__[rubocop]__
* Use 2 (not -13) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->